src/styles/App.style.ts

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+import { device } from "./sizes";
+
+export const AppTitleNonFixed = styled.h1`
+text-align: center;
+`;
+
+export const AppBody = styled.div`
+padding-top: 75px;
+background: linear-gradient(-45deg, #e4b4a4, #e9bbcd, #77c0b0, #7db6aa);
+background-size: 400% 400%;
+-webkit-animation: gradient 15s ease infinite;
+animation: gradient 15s ease infinite;
+display: flex !important;
+flex-direction: column;
+justify-content: center !important;
+`
+
+export const AppContainer = styled.div`
+`
+
+export const PartyTime = styled(AppBody)`
+height: 100%;
+padding-top: 0 !important;
+margin-bottom: 50px;
+display: block;
+text-align: center;
+`;
```

Comment:
All of this is directly copied over from what used to be in `App.tsx` below the imports